web/concrete/src/Asset/AssetList.php

```diff
@@ -1,61 +1,66 @@
+    public $assets = array();
+    public $assetGroups = array();
 
-	public function getRegisteredAssets() {
-		return $this->assets;
-	}
+    public function getRegisteredAssets()
+    {
+        return $this->assets;
+    }
 
-	public static function getInstance() {
-		if (null === self::$loc) {
-			self::$loc = new self;
-		}
-		return self::$loc;
-	}
+    public static function getInstance()
+    {
+        if (null === self::$loc) {
+            self::$loc = new self;
+        }
+        return self::$loc;
+    }
 
-	public function register($assetType, $assetHandle, $filename, $args = array(), $pkg = false) {
-		$defaults = array(
-			'position' => false,
-			'local' => true,
-			'version' => false,
-			'combine' => -1,
-			'minify' => -1 // use the asset default
-		);
-		// overwrite all the defaults with the arguments
-		$args = array_merge($defaults, $args);
+    public function register($assetType, $assetHandle, $filename, $args = array(), $pkg = false)
+    {
+        $defaults = array(
+            'position' => false,
+            'local' => true,
+            'version' => false,
+            'combine' => -1,
+            'minify' => -1 // use the asset default
+        );
+        // overwrite all the defaults with the arguments
+        $args = array_merge($defaults, $args);
 
-		$class = '\\Concrete\\Core\\Asset\\' . Object::camelcase($assetType) . 'Asset';
-		$o = new $class($assetHandle);
+        $class = '\\Concrete\\Core\\Asset\\' . Object::camelcase($assetType) . 'Asset';
+        $o = new $class($assetHandle);
         if ($pkg != false) {
             if (!($pkg instanceof Package)) {
                 $pkg = Package::getByHandle($pkg);
             }
-    		$o->setPackageObject($pkg);
+            $o->setPackageObject($pkg);
+        }
+        $o->setAssetIsLocal($args['local']);
+        $o->mapAssetLocation($filename);
+        if ($args['minify'] === true || $args['minify'] === false) {
+            $o->setAssetSupportsMinification($args['minify']);
+        }
+        if ($args['combine'] === true || $args['combine'] === false) {
+            $o->setAssetSupportsCombination($args['combine']);
         }
-		$o->setAssetIsLocal($args['local']);
-		$o->mapAssetLocation($filename);
-		if ($args['minify'] === true || $args['minify'] === false) {
-			$o->setAssetSupportsMinification($args['minify']);
-		}
-		if ($args['combine'] === true || $args['combine'] === false) {
-			$o->setAssetSupportsCombination($args['combine']);
-		}
-		if ($args['version']) {
-			$o->setAssetVersion($args['version']);
-		}
-		if ($args['position']) {
-			$o->setAssetPosition($args['position']);
-		}
-		$this->registerAsset($o);
-		return $o;
-	}
+        if ($args['version']) {
+            $o->setAssetVersion($args['version']);
+        }
+        if ($args['position']) {
+            $o->setAssetPosition($args['position']);
+        }
+        if ($args['attributes']) $o->setTagAttributes($args['attributes']);
```

Comment:
5. Control Structures
The general style rules for control structures are as follows:

There MUST be one space after the control structure keyword
There MUST NOT be a space after the opening parenthesis
There MUST NOT be a space before the closing parenthesis
There MUST be one space between the closing parenthesis and the opening brace
The structure body MUST be indented once
The closing brace MUST be on the next line after the body
The body of each structure MUST be enclosed by braces. This standardizes how the structures look, and reduces the likelihood of introducing errors as new lines get added to the body.